packages/components/scrollbar/src/scrollbar.vue

```diff
@@ -151,6 +150,19 @@ export default defineComponent({
       { immediate: true }
     )
 
+    watch(
```

Comment:
Should we deep watch or we do something like:

```ts
watch(() => {
  const [_, ourStyle] = style.value
  return {...ourStyle}
}, handler)
```

Reply:
I suspect that this watch would take effect since the `style.value.length` has not been changed, `watch` will not deep watch for us in Vue3.

Reply:
> I suspect that this watch would take effect since the `style.value.length` has not been changed, `watch` will not deep watch for us in Vue3.

style is a `computed` property, so I think it's fine

Reply:
> I suspect that this watch would take effect since the `style.value.length` has not been changed, `watch` will not deep watch for us in Vue3.

I shouldn't listened to the computed property, I changed it to `watch(() => [props.maxHeight, props.height])`